apps/voting/app/src/components/NewVotePanelContent.js

```diff
@@ -31,30 +35,62 @@ class NewVotePanelContent extends React.Component {
     this.props.onCreateVote(this.state.question.trim())
   }
   render() {
-    const { question } = this.state
+    const { question, screenIndex } = this.state
     return (
       <div>
-        <Info.Action title="Votes are informative">
-          They don’t have any direct repercussion on the organization.
-        </Info.Action>
-        <Form onSubmit={this.handleSubmit}>
-          <Field label="Question">
-            <TextInput
-              ref={question => (this.questionInput = question)}
-              value={question}
-              onChange={this.handleQuestionChange}
-              required
-              wide
-            />
-          </Field>
-          <Button mode="strong" type="submit" wide>
-            Begin vote
-          </Button>
-          <Warning>
-            If you are allowed to directly create this vote, you will
-            automatically vote yes for it.
-          </Warning>
-        </Form>
+        <div css="margin: 0 -30px 30px">
+          <TabBar
+            items={['Question', 'Action']}
+            selected={screenIndex}
+            onChange={this.handleTabChange}
+          />
+        </div>
+        {screenIndex === 0 && (
+          <Form onSubmit={this.handleSubmit}>
+            <Field label="Question">
+              <TextInput
+                ref={question => (this.questionInput = question)}
+                value={question}
+                onChange={this.handleQuestionChange}
+                required
+                wide
+              />
+            </Field>
+            <Button mode="strong" type="submit" wide>
+              Begin question
+            </Button>
+            <div css="margin-top: 20px">
+              <Info.Action title="These votes are informative">
+                <div
+                  css={`
+                    margin-top: 5px;
+                    font-size: 14px;
+                  `}
+                >
+                  Questions are used for signaling and don’t have any direct
+                  repercussions on the organization.
+                </div>
+              </Info.Action>
+            </div>
+          </Form>
+        )}
+        {screenIndex === 1 && (
+          <div>
+            <p css="margin: 30px 0 20px">Coming soon!</p>
```

Comment:
Not sure if we decided on keeping this @dizzypaty

Reply:
Mmm... we can keep it if we plan to eventually make it available from here (https://github.com/aragon/aragon-apps/issues/582#issuecomment-443433985). Otherwise I'm fine with removing it! I know this tab still feels a bit weird and empty but I'll definitely rethink this on the next iteration of the voting app.

Reply:
Yeah let's remove the Coming Soon part